### ISSUE TITLE
Provide documentation and example projects

### ISSUE THREAD

**Issue by JayFoxRox** [X123M3-256/TrackRender#4]
This project desperately needs a README.
I found this after only 2 days of working on my own solution; even then it takes a lot of reading of the code to figure out what it does.

I'd like to see (in the README, wiki, or repository website):

- Short introduction about what this does (curve-generation, curve-fitting, sprite-rendering).
- Example input files (ideally a wiki article with a list of all publicly available inputs).
- Example screenshot of the output.
- Documentation on how to build the tool.
- Documentation on how to run this tool.
- Plans for the future of this tool (integration with OpenRCT2? use within https://github.com/OpenRCT2/OpenGraphics?).

**Comment by X123M3-256** [comment]
Hmm, I'll see if I can come up with something. The primary purpose of this project is to support my custom track making project and it wasn't really written with the thought that others might want to use it in mind, so I haven't documented anything. I do have some example input files and models that I could upload, however. I might add them into this repository.

**Comment by JayFoxRox** [comment]
> the primary purpose of this project is to support my custom track making project

Yes, I'm aware. But I feel this has a wider range of applications, like use for https://github.com/Basssiiie/OpenRCT2-Unity or https://github.com/OpenRCT2/OpenGraphics which both have to remodel the original track pieces.
Also others might want to make their own custom tracks in the future.

> I do have some example input files and models that I could upload, however. I might add them into this repository.

I'd suggest to keep them in a separate repository, but link it from this one (README or wiki).
